cli/azd/pkg/project/framework_service_docker.go

```diff
@@ -433,6 +434,16 @@ func (p *dockerProject) packBuild(
 
 		if svc.Language == ServiceLanguageJava {
 			environ = append(environ, "ORYX_RUNTIME_PORT=8080")
+			// Consider it as multi-module project if service path is not same as its project path
```

Comment:
The project path is the file path of project file (that is project root directory), and the service path is the path of corresponding service. Normally in a single-module java project, it should be same because the `pom.xml` file is in the same directory as the project config file. In multi-module java projects, the service path points to the path of sub-module, which is not the project root directory that project config file is in.
So I use this clue to check if it's multi-module java project.